packages/tools/pie-icons-react/scripts/react-icons-gen.js

```diff
@@ -0,0 +1,32 @@
+const svgr = require("@svgr/core");
+const fs = require("fs");
+const pieIcons = require("@justeattakeaway/pie-icons").default;
+const { pascalCase } = require("pascal-case");
+const path = require("path");
+
+const icons = pieIcons.icons;
+
+const ICONS_DIR = process.cwd() + "/src/icons";
```

Comment:
It would be good to keep the generated icon directory similar to the vue package.

We've put generated icons in a folder called `icons`, as we thought that using `src/` may be confusing (mainly as the files are generated). It also means that you can add the individual icon `tsx` files to the package bundle, so that applications could just choose to import an individual component file (if they don't use a build tool that allows them to tree-shake for instance).

Maybe can catch up later when you're online about this one?